clrfeatures.cmake

```diff
@@ -25,3 +25,9 @@ endif(NOT DEFINED FEATURE_DBGIPC)
 if(NOT DEFINED FEATURE_INTERPRETER)
   set(FEATURE_INTERPRETER 0)
 endif(NOT DEFINED FEATURE_INTERPRETER)
+
+if(NOT DEFINED FEATURE_NI_BIND_FALLBACK)
+  if(CLR_CMAKE_TARGET_ARCH_AMD64 OR CLR_CMAKE_TARGET_ARCH_ARM64)
```

Comment:
This "if" check should be NOT of the existing "if" to account for my comment at https://github.com/dotnet/coreclr/pull/11485/files#r115656309.